sdk/Managed/src/Microsoft.WindowsAzure.MobileServices/Table/Sync/Queue/OperationQueue.cs

```diff
@@ -149,6 +149,32 @@ public Task<IDisposable> LockItemAsync(string id, CancellationToken cancellation
             }
         }
 
+        public virtual async Task<bool> UpdateAsync(string id, long version, JObject item)
+        {
+            try
+            {
+                MobileServiceTableOperation op = await GetOperationAsync(id);
+                if (op == null || op.Version != version)
+                {
+                    return false;
+                }
+
+                // Change the operation state back to pending since this is a newly updated operation without any conflicts
+                op.State = MobileServiceTableOperationState.Pending;
```

Comment:
The original idea of this flag, was is it possible the server has seen part of this request.  In this case, the answer to that is clearly Yes (since this is a conflict/etc).  I can see both ways here, it should be documented clearly that this resets our tracking of the op in the public docs on the method.